app/tests/test_queries.py

```diff
@@ -482,6 +484,524 @@ def test_browse_data_with_consignment_filter(self, app):
+        self, client: FlaskClient
+    ):
+        """
+        Given multiple File objects in the database and consignment transfer complete date
+            that matches all of them
+        When browse_data is called with date_from and date_to
+        Then it returns a list containing multiple dictionary for the matching record with
+            date in between the range
+        """
+        files = create_multiple_test_records()
+
+        date_range = {"date_from": "01/02/2023", "date_to": "28/02/2023"}
+        result = browse_data(page=1, per_page=per_page, date_range=date_range)
+
+        assert result.items == [
+            (
+                files[4].file_consignments.consignment_bodies.BodyId,
+                "testing body5",
+                files[4].file_consignments.consignment_series.SeriesId,
+                "test series5",
+                "15/02/2023",
+                1,
+                1,
+            ),
+            (
+                files[5].file_consignments.consignment_bodies.BodyId,
+                "testing body6",
+                files[5].file_consignments.consignment_series.SeriesId,
+                "test series6",
+                "15/02/2023",
+                1,
+                1,
+            ),
+            (
+                files[6].file_consignments.consignment_bodies.BodyId,
+                "testing body7",
+                files[6].file_consignments.consignment_series.SeriesId,
+                "test series7",
+                "15/02/2023",
+                1,
+                1,
+            ),
+            (
+                files[7].file_consignments.consignment_bodies.BodyId,
+                "testing body8",
+                files[7].file_consignments.consignment_series.SeriesId,
+                "test series8",
+                "15/02/2023",
+                1,
+                1,
+            ),
+            (
+                files[8].file_consignments.consignment_bodies.BodyId,
+                "testing body9",
+                files[8].file_consignments.consignment_series.SeriesId,
+                "test series9",
+                "15/02/2023",
+                1,
+                1,
+            ),
+        ]
+
+    def test_browse_data_with_date_from_and_to_filter_no_result(
+        self, client: FlaskClient
+    ):
+        """
+        Given multiple File objects in the database and consignment transfer complete date
+            that matches all of them
+        When browse_data is called with date_from and date_to
+        Then if date range not matched it returns empty list containing multiple dictionary
+        """
+        create_multiple_test_records()
+
+        date_range = {"date_from": "01/03/2023", "date_to": "31/05/2023"}
+        result = browse_data(page=1, per_page=per_page, date_range=date_range)
+
+        assert result.items == []
+
+    def test_browse_data_date_from_filter_exception_raised(self, app, caplog):
```

Comment:
If you are going to test `get_date_range` can you move these tests to their own file, since they are not testing queries.

Reply:
date_range tests have been removed from test_queries.py and added a in a new file test_date_range.py